### PR TITLE
[wni] Fix broken go.sum

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
As of now, go.sum on master branch is broken.
This commit should fix it.

/cc @aravindhp @vinaykns

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
> Please update your PR and commit header to reflect what you are fixing. i.e. [wni] Fix broken go.sum. Please say which PR introduced the issue and how you are fixing it in the commit and PR message.

PR #24 introduced the problem. The problem is not limited to go.mod and go.sum within wni directory as we initially thought, the go.mod in wmco root is also effected because of the #24 changes. Fixed it using `go mod verify` in both the locations. 

Updated the commit message to reflect it.

File: go.mod [hunk 1]
```diff
@@ -12,16 +12,9 @@ replace (
 )
 
 require (
-	github.com/Azure/azure-sdk-for-go v33.4.0+incompatible // indirect
```
Comment (ravisantoshgudimetla) [inline go.mod]: These are not needed in go.sum of wmco dir.

**Comment by sebsoto** [comment]
/lgtm
Ran `make all` and go.sum is the same

**Comment by sebsoto** [comment]
Holding in case other reviewer is needed

**Pull request merged**